contrib/registry/etcd/etcd.go

```diff
@@ -80,6 +82,15 @@ func New(address string, option ...Option) gsvc.Registry {
 	if password != "" {
 		cfg.Password = password
 	}
+	if option != nil {
+		if option[0].DialTimeout > 0 {
+			cfg.DialTimeout = option[0].DialTimeout
+		}
+	} else {
+		cfg.DialTimeout = time.Second * 5
+	}
+	cfg.AutoSyncInterval = time.Second
```

Comment:
- Please make this attribute `AutoSyncInterval` as public attribute of `Option` user can customize. You can make it default value as `time.Second` if user does not provide.
- Please define the default values as constants.